### PR TITLE
Fix for Frontend issues #1106

### PR DESCRIPTION
**Pull request by prince-7**
Fix for the issue described in #1106
## Description

The frontend issues have been fixed.

## Related Issue
#1106 

## Reviewers

@sharmamohit123 @viyatb @7a 

## Screenshots (if appropriate):

1. Fixed "1.Breaking GIFs in Dashboard."
workers
2. Fixed "2. In the Report page, the Side panel should fit the screen vertically." and "3. When no Plugin is running, the error message should be replaced with "No plugins found" message."
noplugins

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Other

### Checklist:

- [x] My code follows the code style (modified PEP8) of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: owtf/webapp/src/containers/Report/Collapse.js [hunk 1]
```diff
@@ -190,9 +190,7 @@ export default class Collapse extends React.Component {
           justifyContent="center"
           width={700}
         >
-          <Paragraph margin={40}>
-            Something went wrong, please try again!
-          </Paragraph>
+          <Paragraph margin={40}>No plugins found</Paragraph>
```
Comment (sharmamohit123) [inline owtf/webapp/src/containers/Report/Collapse.js]: This is not required.

File: owtf/webapp/src/containers/Report/SideFilters.js [hunk 2]
```diff
@@ -257,7 +257,12 @@ export class SideFilters extends React.Component {
 
         {/* Type Filter starts*/}
         <Strong marginBottom={10}> Plugin Type</Strong>
-        <Tablist display="flex" flexDirection="column" width={200}>
+        <Tablist
+          display="flex"
+          flexDirection="column"
+          width={200}
+          marginBottom="100%"
```
Comment (sharmamohit123) [inline owtf/webapp/src/containers/Report/SideFilters.js]: A small margin still remains.
Screenshot from 2021-03-20 04-26-30

File: owtf/webapp/src/containers/Report/Collapse.js (hunk 1 above)
Comment (prince-7) [reply owtf/webapp/src/containers/Report/Collapse.js]: Restored

File: owtf/webapp/src/containers/Report/SideFilters.js (hunk 2 above)
Comment (prince-7) [reply owtf/webapp/src/containers/Report/SideFilters.js]: fixed

**Comment by prince-7** [comment]
Changes made please check!

**Comment by sharmamohit123** [comment]
@prince-7 thanks for the changes. @viyatb pls merge this

**Pull request merged**